websites/docs-smoke-test/src/data/navigation.yaml

```diff
@@ -1,71 +1,31 @@
-- chapter-title: Basic Content
+- chapter-title: Views
   pages:
-    - title: Text
-      path: /smoke-tests/text/
-    - title: Headings and Index Nav
-      path: /smoke-tests/right-nav-submenu/
-    - title: Links
-      path: /smoke-tests/links/
-    - title: Link Target Page
-      path: /smoke-tests/links/link-target
-    - title: Subtitle and Lead
-      path: /smoke-tests/lead-section/
-    - title: Tables
-      path: /smoke-tests/table/
-    - title: Blockquotes
-      path: /smoke-tests/blockquotes/
-    - title: Horizontal Rule
-      path: /smoke-tests/horizontal-rule/
-    - title: Lists
-      path: /smoke-tests/lists/
-    - title: Definition Lists
-      path: /smoke-tests/definition-lists/
-    - title: Empty Page
-      path: /smoke-tests/empty-page/
-    - title: Page with little content
-      path: /smoke-tests/page-with-little-content/
-- chapter-title: Custom Elements
+    - title: Empty
+      path: /views/empty
```

Comment:
FYI: the history of the empty page was that it was intended to be a VRT because there were layout bugs when there was no content or extremely short content (less than a line length).  I added it to find regressions over the positioning in big screen widths.

Reply:
Yeah, it’s currently tested by the e2e tests